README.md

```diff
@@ -1 +1,37 @@
-# angular-what-is-a-directive-readme
\ No newline at end of file
+# What is a Directive?
+
+## Overview
+
+You might not have noticed, but we've used directives already in our previous labs. Directives are essential to learning Angular, so hold in tight!
+
+## Objectives
+
+- Describe what Directives are
+
+## Directives, deep down
+
+Directives can be compared to components. They are essentially "markers" on a HTML element that tell Angular to attach a specific behaviour to the HTML element - either something small like a click event, or actually completely transforming the DOM node to display a list of data.
+
+Directives can be either an actual HTML element or an attribute on a HTML element.
+
+```html
+<my-directive></my-directive>
+
+<!-- these are the same -->
+
+<div my-directive></div>
+```
+
+However, not all directives can be used in both ways - some are restricted to being just an attribute or just an element.
```

Comment:
Is this specific to each directive? Is there a way to know whether directives can be used as an element or as an attribute? If so, include a brief explanation of the difference here. Otherwise, you could just include a note letting the student know that we'll discuss the differences in future lessons.